<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>435</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>427</width>
    <height>435</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>427</width>
    <height>435</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>计数器</string>
  </property>
  <property name="windowIcon">
   <iconset resource="re.qrc">
    <normaloff>:/res/jsq.jpg</normaloff>:/res/jsq.jpg</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">#frame { 
	border-image: url(:/res/siverbg.png);
}

#frame * {
	border-image:url();
}
#n0Buttom:pressed{
	border-image: url(:/res/siverbg.png);
}</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-10</y>
     <width>451</width>
     <height>471</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="n7Buttom">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/res/d3btn-a.gif);
color: rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;
</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="n8Buttom">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>200</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/res/d3btn-a.gif);
color: rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;
</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="n9Buttom">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>200</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/res/d3btn-a.gif);
color: rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;
</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="n4Buttom">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/res/d3btn-a.gif);
color: rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;
</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="n5Buttom">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>250</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/res/d3btn-a.gif);
color: rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;
</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="n6Buttom">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>250</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/res/d3btn-a.gif);
color: rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;
</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="n1Buttom">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/res/d3btn-a.gif);
color: rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;
</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="n2Buttom">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>300</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/res/d3btn-a.gif);
color: rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;
</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="n3Buttom">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>300</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/res/d3btn-a.gif);
color: rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;
</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="n0Buttom">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/res/d3btn-a.gif);
color: rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;
</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="npButtom">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>350</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/res/d3btn-a.gif);
color: rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;
</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="minusButtom">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>350</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/res/d3btn-a.gif);
color: rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;
</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="multiButtom">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>200</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/res/d2btn-a.gif);
color: rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;
</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="subButtom">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>250</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/res/d2btn-a.gif);
color: rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;
</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addButtom">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>300</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/res/d2btn-a.gif);
color: rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;
</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="equalButtom">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>350</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;
border-image: url(:/res/d4btn-a.gif);</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="divisionButtom">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>150</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/res/d2btn-a.gif);
color: rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;
</string>
    </property>
    <property name="text">
     <string>÷</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cleanButtom">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>150</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/res/d0btn-a.gif);
color: rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;
</string>
    </property>
    <property name="text">
     <string>清屏</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delButtom">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/res/d0btn-a.gif);
color: rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;
</string>
    </property>
    <property name="text">
     <string>退格</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quitButtom">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/res/d1btn-a.gif);
color: rgb(255, 255, 255);
font: 13pt &quot;微软雅黑&quot;;
</string>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QLabel" name="displayLabel">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>30</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 21pt &quot;微软雅黑&quot;;
text-align:right;
</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="re.qrc"/>
 </resources>
 <connections/>
</ui>
